docs/developer/stock-allocation.mdx

```diff
@@ -43,13 +45,32 @@ classDiagram
     class Warehouse{
       click_and_collect_option: String
     }
+    class Channel{
+      allocationStrategy: String
+    }
 ```
 
+### Allocation strategies
+
+The allocation strategies allow defining the rule that is used during the stock
+allocation. At this moment two possible options are available:
+
+- Prioritize warehouses by sorting order (`PRIORITIZE_SORTING_ORDER`) - allocate stocks
+  according to the warehouses' order within the channel. If stock is insufficient,
```

Comment:
It would be useful to link `warehouses' order` to 
https://saleor-docs-mguwbmz3t-saleorcommerce.vercel.app/docs/3.x/developer/channels#reorder-warehouses-within-channels

Reply:
Hm, but I linked this in the last sentence.

>  To change the warehouses' order within the channel, use `channelReorderWarehouses` mutation.